Assets/MixedRealityToolkit.SDK/Inspectors/UX/Interactable/InteractableInspector.cs

```diff
@@ -678,6 +699,66 @@ protected void SetupEventOptions()
             eventOptions = InteractableEvent.GetEventTypes();
         }
 
+        protected string[] GetEventList()
+        {
+            return new string[] { };
+        }
+
+        #endregion Events
+
+        #region PopupUtilities
+
+        /// <summary>
+        /// Look for speech commands in the MRTK Speech Command profile
+        /// </summary>
+        /// <param name="keywords"></param>
+        /// <returns></returns>
+        protected bool TryGetSpeechKeywords(out string[] keywords)
+        {
+            List<string> keys = new List<string>();
+            
+            if (MixedRealityToolkit.Instance && (MixedRealityToolkit.Instance.ActiveProfile.IsInputSystemEnabled ||
+                MixedRealityToolkit.Instance.ActiveProfile.InputSystemProfile.SpeechCommandsProfile != null ||
+                MixedRealityToolkit.Instance.ActiveProfile.InputSystemProfile.SpeechCommandsProfile.SpeechCommands.Length > 0))
+            {
+                int keywordCount = MixedRealityToolkit.Instance.ActiveProfile.InputSystemProfile.SpeechCommandsProfile.SpeechCommands.Length;
+
+                for (var i = 0; i < keywordCount; i++)
+                {
+                    keys.Add(MixedRealityToolkit.Instance.ActiveProfile.InputSystemProfile.SpeechCommandsProfile.SpeechCommands[i].Keyword);
+                }
+
+                keys.Insert(0, "None");
```

Comment:
What do we do if the user selects "none" for the speech commands here?

Reply:
We set the VoiceCommand property to an empty string. There's code in Interactable and the inspector that checks for an empty string to see if listening for voice commands is active or not.